### DISCUSSION TITLE
include tailwind css in bundle js

### DISCUSSION THREAD

**Discussion by musashiM82** [tailwindlabs/tailwindcss#8255] [Help]
Hi 
How could I include tailwind css in bundle js ? 

this is the an example with vue 3 and tailwind 3 https://github.com/musashiM82/vue-webpack.

running `npm run build`, it creates 3 files:
* app.js                  
* ABOUTPAGE.js
* app.6cba1802.css

I want to include app.6cba1802.css into app.js, so the results should be:

* app.js                  
* ABOUTPAGE.js

Thanks

**Comment by ugurkellecioglu** [comment]
It's been 2 years but I need the same thing, did you find any solution to this @musashiM82 ?
